core/src/main/scala/caliban/Rendering.scala

```diff
@@ -91,6 +94,11 @@ object Rendering {
     case Some(value)            => if (value.contains("\n")) s"""\"\"\"$value\"\"\" """ else s""""$value" """
   }
 
+  private def renderSpecifiedBy(specifiedBy: Option[String]): String = specifiedBy match {
```

Comment:
nit/question: do we prefer to `fold` or `match` in these cases?

Reply:
Good point, will use fold. Done.